cmd/podman/spec.go

```diff
@@ -351,6 +351,22 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Seccomp = nil
 	}
 
+	// Add Secret Mounts
+	mountsFiles := []string{libpod.OverrideMountsFile, config.DefaultMountsFile}
+	for _, file := range mountsFiles {
+		secretMounts, err := libpod.SecretMounts(file, config.MountLabel, config.WorkDir)
```

Comment:
I don't think this should bin in libpod but in libpod/pkg/secrets (That is what my patch did).

Reply:
We should be doing this in containers_internal.go not here.  Since we want to keep the contents even if we do a stop/start.